test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -615,7 +616,8 @@ protected bool CheckEqualityFromPathsCore(string relPath, string basePath, strin
 
         private static void GetNumbersFromFile(ref string firstString, ref string secondString, decimal precision)
         {
-            Regex _matchNumer = new Regex(@"\b[0-9]+\.?[0-9]*\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+            string decimalSeparator = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
+            Regex _matchNumer = new Regex(@"\b[0-9]+\" + decimalSeparator + "?[0-9]*\b", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
Does a `,` need a backslash before it like a decimal point `.` does?  I see you left the backslash in.

Reply:
For ```,```  the backslash wont have any effect on the output of the regular expression.